spring-cloud-commons/src/main/java/org/springframework/cloud/client/loadbalancer/RetryLoadBalancerInterceptor.java

```diff
@@ -58,6 +58,9 @@
 
 	private final ReactiveLoadBalancer.Factory<ServiceInstance> loadBalancerFactory;
 
+	private LoadBalancerPropertiesFactory factoryProperties;
```

Comment:
Please rename the field to `propertiesFactory`.

Reply:
Changed